jujugui/static/gui/src/app/components/terminal/test-terminal.js

```diff
@@ -94,10 +94,13 @@ describe('Terminal', () => {
         }}
         WebSocket={websocket} />
     );
+    // Check that fit is called after receiving the first PS1.
+    component.term.fit = sinon.stub(); // eslint-disable-line
```

Comment:
Why the eslint disable line?

Reply:
We have a custom eslint rule that doesn't allow 'fit' in tests because that is the function call for a single test run.